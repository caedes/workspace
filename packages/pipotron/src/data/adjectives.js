export default [
  'présente',
  'actuelle',
  'générale',
  'induite',
  'conjoncturelle',
  'observée',
  'contextuelle',
  'de ces derniers temps',
  "de l'époque actuelle",
  'intrinsèque',
  'que nous constatons',
];
