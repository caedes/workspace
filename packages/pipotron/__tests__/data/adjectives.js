import adjectives from '../../src/data/adjectives';

describe('adjectives', () => {
  it('returns the adjectives', () => {
    expect(adjectives).toMatchSnapshot();
  });
});
