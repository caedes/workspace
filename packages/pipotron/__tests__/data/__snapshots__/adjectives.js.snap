// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`adjectives returns the adjectives 1`] = `
Array [
  "présente",
  "actuelle",
  "générale",
  "induite",
  "conjoncturelle",
  "observée",
  "contextuelle",
  "de ces derniers temps",
  "de l'époque actuelle",
  "intrinsèque",
  "que nous constatons",
]
`;
